src/themes/components/textarea.ts

```diff
@@ -0,0 +1,26 @@
+import type {
+  SystemStyleInterpolation,
+  SystemStyleObject,
+} from '@chakra-ui/theme-tools';
+
+import Input from './input';
+
+const baseStyle: SystemStyleObject = {
+  px: 'lg',
+  pt: 'md',
```

Comment:
So the padding would be asymmetric? Is that intended?

Reply:
Thanks for double-checking 👍🏻

Reply:
After checking with Enrique:
We keep the padding as we have. The reasons are:
1. On the bottom we don't have padding because is a textarea and the content could exceed and scroll.
2. The top padding is smaller than the side paddings to equalize the visual distance of the text.

Reply:
Thanks for checking 👍🏻